src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -31,19 +29,14 @@ public void askForMove() {
         output.println(language.promptForMove());
     }
 
-    public Moves getMoveThree(String gameMode) {
+    public Moves getMove() {
         String move = null;
-        if (gameMode.equals("1")) {
-            try {
-                move = input.readLine();
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
-        } else {
-            move = Computer.playMoveTwo();
+        try {
+            move = input.readLine();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Is that a deliberate choice to printStackTrace?

Wouldn't it easier to console log your own error message here? May be print something in order to fix/evoid this error again

Reply:
Thank you @fabientownsend, I will look into this!